components/style/values/specified/box.rs

```diff
@@ -7,15 +7,429 @@
 use Atom;
 use cssparser::Parser;
 use parser::{Parse, ParserContext};
+#[cfg(feature = "servo")]
+use properties::{longhands, PropertyDeclaration};
 use std::fmt;
 use style_traits::{ParseError, ToCss};
 use values::CustomIdent;
 use values::KeyframesName;
+#[cfg(feature = "servo")]
+use values::computed::Context;
 use values::generics::box_::AnimationIterationCount as GenericAnimationIterationCount;
 use values::generics::box_::VerticalAlign as GenericVerticalAlign;
 use values::specified::{AllowQuirks, Number};
 use values::specified::length::LengthOrPercentage;
 
+// FIXME(emilio): Why does this reinvent the wheel, again?
+#[allow(non_camel_case_types)]
```

Comment:
Why is this needed?

Reply:
I think after moving this `enum` into specified module, we don't need this anymore.
Besides, I think even in the old way, maybe we don't need it either?

Reply:
Yeah, probably just leftover.

Reply:
And now it uses `derive(Parse)` the `FIXME` can go away too :)